mls-rs-codec/src/lib.rs

```diff
@@ -121,10 +121,7 @@ pub trait MlsDecode: Sized {
     fn mls_decode(reader: &mut &[u8]) -> Result<Self, Error>;
 }
 
-impl<T> MlsDecode for Box<T>
-where
-    T: MlsDecode + ?Sized,
-{
+impl<T: MlsDecode> MlsDecode for Box<T> {
```

Comment:
Not sure about this one?

Reply:
Removing `?Sized` is ok (and non-breaking) because there's already `T: MlsDecode` and `MlsDecode` has `Sized` as a super-trait, meaning any implementor of `MlsDecode` has to be `Sized`.

Reply:
New clippy found this